examples/vision/md/randaugment.md

```diff
@@ -0,0 +1,421 @@
+# RandAugment for Image Classification for Improved Robustness
```

Comment:
```suggestion
# Automated data augmentation with RandAugment for image classification
```

(This way users who don't know RandAugment can click on the example because it's for "data augmentation that does X". Automated data aug. is the main 🍖 , if I understood the paper correctly - ML for ML.)

Reply:
I explained it above.